recipes/metis/all/conandata.yml

```diff
@@ -5,12 +5,28 @@ sources:
 patches:
   "5.1.1":
     - patch_file: "patches/001-add-gklib-system-cmake.patch"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+      patch_description: "Add gklib system cmake"
     - patch_file: "patches/002-support-pure-cmake.patch"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+      patch_description: "Support pure CMake"
     - patch_file: "patches/003-remove-programs.patch"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+      patch_description: "Remove building of programs"
     - patch_file: "patches/004-use-conan-gklib.patch"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+      patch_description: "Use conan gklib"
     - patch_file: "patches/005-fix-install.patch"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+      patch_description: "Fix runtime install"
+    - patch_file: "patches/006-require-newer-cmakelists.patch"
```

Comment:
@joakimono why is this needed? I saw upstream is requiring 2.8, is it related to using the `CMAKE_POLICY_DEFAULT_CMP0042` policy?

Reply:
I changed it due to remove another policy warning CMP0054 regarding quoted variables. I tried to suppress this like is done  with CMP0042 above, but for some reason it had no effect. The warning is triggered by the generated `cmakedeps_macros.cmake:33`. I assumed that changing the minimum required cmake in the upstream did not matter much, since the recipe requires 3.15 anyway. It should be noted that the metis CMakeLists.txt files seem pretty standard to me and do not use any deprecated 2.8 functionality.